freqtrade/rpc/telegram.py

```diff
@@ -20,6 +20,9 @@
 logger.debug('Included module rpc.telegram ...')
 
 
+MAX_TELEGRAM_MESSAGE = 4096
```

Comment:
> also maybe we want to include it in RPC as a cross messaging constant instead of here.

I don't think that makes sense - this limit applies to telegram only - other methods will have completely different length limits (and even if it's the same, each service can change it independently)...